ctx.go

```diff
@@ -526,6 +525,15 @@ func (c *Ctx) Get(key string, defaultValue ...string) string {
 	return defaultString(c.app.getString(c.fasthttp.Request.Header.Peek(key)), defaultValue)
 }
 
+// GetRespHeader returns the HTTP response header specified by field.
+// Field names are case-insensitive
+// Returned value is only valid within the handler. Do not store any references.
+// Make copies or use the Immutable setting instead.
+func (c *Ctx) GetRespHeader(key string, defaultValue ...string) string {
```

Comment:
we need a part in the docs repository for this new method

Reply:
How about this?

Reply:
Sorry, I am not clear, do you mean I should add the annotation for this new method in https://github.com/gofiber/docs/blob/master/api/ctx.md?

Reply:
yes and please add a unitttest for the method

Reply:
👌